sdk/src/wallet/account/builder.rs

```diff
@@ -35,7 +35,7 @@ pub struct AccountBuilder {
     #[cfg(feature = "events")]
     event_emitter: Arc<Mutex<EventEmitter>>,
     #[cfg(feature = "storage")]
-    storage_manager: Arc<Mutex<StorageManager>>,
+    storage_manager: Arc<tokio::sync::Mutex<StorageManager>>,
```

Comment:
Just remove `#[cfg(feature = "events")]` in L9 instead?

Reply:
but then you get and unused import if you compile without either of the features as it's only used there. I missed to qualify the `Mutex` in line 36 tho.